src/core/utilities/auto-match-logic.ts

```diff
@@ -132,7 +144,7 @@ export function findMostCommonShowName(showList: Array<PathDetails | null>): str
     return '';
   }
 
-  const showNameMap = reduce(showList, (acc, show) => {
+  const showNameMap = showList.reduce((acc, show) => {
```

Comment:
what is the point of replacing lodash ? It has to be consistent throughout the project.

Reply:
I can add lodash back.